### DISCUSSION TITLE
Roll a generic attack as GM

### DISCUSSION THREAD

**Discussion by designvoid** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#461] [Q&A] [answered]
How do I roll say 6 dice as a GM and get it to highlight swords and skulls? I don't want it to be as part of a monster attack - say its a magical trap or the like - maybe a FEAR attack and its 4dice and swords cause damage to WITS?

**Comment by aMediocreDad** [comment, marked as answer]
`/fblr 4db` in chat for example.

**Reply by designvoid** [reply]
Absolutely perfect - i just knew it would be something dead simple! Thanks for the speedy answer and awesome work!
